Pipfile

```diff
@@ -9,7 +9,7 @@ jsonschema = "*"
 jupyter = "*"
 pytest = "*"
 pyld = "*"
-pandas = "*"
```

Comment:
I added pandas back in and the 'GDC to CCDH conversion' notebook worked. I think we had some discussion about numpy (via pandas) causing a conflict with something else. Is that still a problem? pipenv resolved everything for me.

Reply:
Oh, nice! I bet I just messed up my Pipfile.lock is some strange way. If you can run:
```
$ pipenv --rm # Delete the existing Pipenv
$ pipenv install # Install a new Pipenv using the Pipfile
                 # You'll need `--dev` if there are development packages here, but there shouldn't be?
$ pipenv run jupyter notebook # Run the Jupyter Notebook
```
Then all should be good. You might need to recommit the Pipfile.lock after doing that.

Could you try that and report back? If it works, then feel free to overwrite my commit.

> The rdflib-jsonld package has been integrated into rdflib as of rdflib==6.0.0.

Neat. I guess that means you can delete rdflib-jsonld from the Pipfile as well!